frontend/ui/widget/textboxwidget.lua

```diff
@@ -1122,6 +1134,7 @@ function TextBoxWidget:free(full)
         self.cursor_restore_bb:free()
         self.cursor_restore_bb = nil
     end
+    self.line_num_to_image = nil
```

Comment:
Bad idea: thumbnails in Wikipedia results and full articles are not shown after this (the room for the image is still there, but not image displayed, and long press in the hold select texts around).
Remove this line and all works again.
(Except for the numerous segfault I get when existing KOreader :/)